specification/2.0/README.md

```diff
@@ -2470,14 +2470,14 @@ A set of parameter values that are used to define the metallic-roughness materia
    
 ### material.normalTexture
 
-A tangent space normal map. Each texel represents the XYZ components of a normal vector in tangent space. <TODO: Add more explaination on how XYZ is calculated>.
+A tangent space normal map. The texture contains RGB components in linear space. Each texel represents the XYZ components of a normal vector in tangent space. Red [0 to 255] maps to X [-1 to 1]. Green [0 to 255] maps to Y [-1 to 1]. Blue [128 to 255] maps to Z [0 to 1]. The normal vectors use OpenGL conventions where +X is right and +Y is up. +Z points toward the viewer.
 
 * **Type**: [`material.normalTextureInfo`](#reference-material.normalTextureInfo)
 * **Required**: No
 
 ### material.occlusionTexture
 
-The occlusion map is a greyscale texture, with white indicating areas that should receive full indirect lighting and black indicating no indirect lighting. <Todo: sRGB ? Color Space>.
+The occlusion map is a greyscale texture in sRGB color space, with white indicating areas that should receive full indirect lighting and black indicating no indirect lighting. If an alpha channel is present, it is ignored.
```

Comment:
> greyscale texture in sRGB color space

Only some GPU (e.g., PowerVR 6 or Adreno 400) support hardware sRGB sampling of one-channel textures. 
While client could unpack grey-scale PNG to three-component texels, it's not very GPU memory efficient.

To get accurate values, client without such h/w support would need to sample, filter and un-sRGB texels manually in shaders (e.g., via `texelFetch` in GLSL ES 3.0). Other option is to use linear hardware filtering on sRGB data which could produce slightly incorrect result.

> If an alpha channel is present, it is ignored.

In general case, texture could have from one to four components:
- `LUMINANCE`: `(L, L, L, 1)`
- `LUMINANCE_ALPHA`: `(L, L, L, A)`
- `RED`: `(R, 0, 0, 1)`
- `RG`: `(R, G, 0, 1)`
- `RGB`: `(R, G, B, 1)`
- `RGBA`: `(R, G, B, A)`

What channel should shaders use (assume RED since it's always populated)?